Sources/MapboxMaps/Style/Types/ExpressionArgumentBuilder.swift

```diff
@@ -90,3 +91,16 @@ extension UIColor: ExpressionArgumentConvertible {
         return [.string(StyleColor(self).rgbaString)]
     }
 }
+
+extension GeoJSONObject: ExpressionArgumentConvertible {
+    public var expressionArguments: [Expression.Argument] {
+        do {
+            let data = try JSONEncoder().encode(self)
+            let expression = try JSONDecoder().decode(Expression.Argument.self, from: data)
+            return [expression]
+        } catch {
+            // Throwable getters are new to Swift 5.5, so we’ll have to live with a fatal error for now.
+            fatalError("Unable to convert GeoJSON object to expression argument: \(error)")
```

Comment:
Alternatively, we could turn `ExpressionArgumentConvertible.expressionArguments` into a method until we can require Swift 5.5 to get throwable getters. Changing the property to a method would be a breaking change.

Reply:
If Turf fails to round-trip a GeoJSONObject for some reason, that would be an internal error instead of programmer error, so throwing an assertion would probably be more appropriate than throwing an error anyways.